project-base/tests/FrontendApiBundle/Test/GraphQlTestCase.php

```diff
@@ -139,4 +139,40 @@ protected function getFullUrlPath(string $uri): string
     {
         return $this->overwriteDomainUrl . $uri;
     }
+
+    /**
+     * @param array $response
+     * @return array
+     */
+    protected function getErrorsOfResponse(array $response): array
+    {
+        $this->assertArrayHasKey('errors', $response);
+        return $response['errors'];
+    }
+
+    /**
+     * @param array $response
+     * @return array
+     */
+    protected function getErrorsExtensionValidationOfResponse(array $response): array
```

Comment:
Please split this method in two, one checker that does asserts and second getter, that returns data from response